### PR TITLE
Add IconSelector component

### PR DESCRIPTION
**Pull request by hissy**
FontAwesome icon selector for feature block

https://user-images.githubusercontent.com/514294/134054305-f1b36dd9-200d-4962-9801-f1baff2acd65.mov

### CODE REVIEW

File: stories/basics/iconSelector.stories.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    </div>
+    <div class="form-group">
+        <icon-selector name="fa-icon-selected" selected="fas fa-address-book" />
+    </div>
+    <div class="form-group">
+        <icon-selector name="fa-icon-custom-title" title="Choose One" empty-option-label="Unselect" />
+    </div>
+</div>`
+
+window.$ = window.jQuery = jQuery
```
Comment (mlocati) [inline stories/basics/iconSelector.stories.js:29]: Are we sure this is wise?

Comment (hissy) [reply stories/basics/iconSelector.stories.js:29]: Yeah, I believe this is fine, it's only for storybook preview. But I'm trying to link this pull request to my local concrete5 repository.

**Comment by aembler** [comment]
If you want to make this work in the release candidate you should probably open this up against 9.0rc instead; also can you fix the JS/CSS style linting errors? You can test it locally by running `npm run lint`

**Comment by hissy** [comment]
@aembler Changed the code base to `9.0rc` . The last commit is the result of `npm run lint:fix:js`

**Pull request merged**